snapcraft.yaml

```diff
@@ -898,12 +882,14 @@ parts:
       - -Dmaintainer-mode=true
       - -Dbuild-documentation=false
       - -Dbuild-demos=false
+    build-packages:
+      - graphviz
```

Comment:
@sergio-costas graphviz is required to build docs, and the dependency is turned on by maintainer mode (see https://gitlab.gnome.org/GNOME/gtkmm/-/blob/master/meson.build?ref_type=heads#L93) is it necessary to have maintainer mode on?

Reply:
Honestly, I don't know. I suspect that it is something that was already there before I arrived. @kenvandine ?

Reply:
I don't recall, but it would be better to disable maintainer mode wherever possible

Reply:
by having second look at `meson.build`, maintainer mode is turned on whenever a git build is performed, which is always the case so it doesn't make a lot of sense to turn it off and hence `graphviz` is a must

Reply:
The point is... what is "maintainer mode"? I've been checking the sources, and it seems that enabling it adds several files with C++ classes for each Gtk widget. But I presumed that they should be autogenerated from gobject introspection...

Reply:
Ok, according to Emanuel Bassi, the maintainer mode rebuilds the wrapper files, which I think that is a good idea...